src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -0,0 +1,246 @@
+  <div id="amalgamating-businesses">
+    <v-btn
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="btn-outlined-primary"
+      :disabled="addAmalgatingBusinessDisabled"
+      @click="onAddBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Business</span>
+    </v-btn>
+
+    <v-btn
+      v-if="isRoleStaff"
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="ml-2 btn-outlined-primary"
+      :disabled="addAmalgatingForeignBusinessDisabled"
+      @click="onAddForeignBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Foreign Business</span>
+    </v-btn>
+
+    <!-- Add an Amalgamating Business button clicked -->
+    <v-expand-transition>
+      <v-card
+        v-if="isAddingAmalgamatingBusiness"
+        flat
+        class="section-container mt-4"
+      >
+        <v-row
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Add a Business Registered in BC</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="8"
+            class="ml-8"
+          >
+            <span>Enter the name or the incorporation number of the registered BC business
+              to add to this application.
+            </span>
+            <BusinessLookup
+              :showErrors="false"
+              :businessLookup="initialBusinessLookupObject"
+              :BusinessLookupServices="BusinessLookupServices"
+              label="Business Name or Incorporation Number"
+              @setBusiness="setAmalgamatingBusiness($event)"
+            />
+            <v-row
+              class="justify-end mr-0 mt-2"
+            >
+              <v-btn
+                id="app-cancel-btn"
+                large
+                outlined
+                color="primary"
+                @click="addAmalgamatingBusinessCancel()"
+              >
+                <span>Cancel</span>
+              </v-btn>
+            </v-row>
+          </v-col>
+        </v-row>
+      </v-card>
+    </v-expand-transition>
+
+    <!-- Add an Amalgamating Foreign Business button clicked -->
+    <v-expand-transition>
+      <v-card
+        v-if="isAddingAmalgamatingForeignBusiness"
+        flat
+        class="section-container mt-4"
+      >
+        <v-row
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Add a Foreign Business</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="8"
+            class="ml-8"
+          >
+            <span>**TODO**</span>
+          </v-col>
+
+          <!-- extra column is for possible action button -->
+        </v-row>
+      </v-card>
+    </v-expand-transition>
+    <v-row class="mt-4 ml-1">
+      <ul>
+        Amalgamating Businesses: <br><br>
```

Comment:
This is all debug code here (can be removed at any point). This is just to show what happens whenever a business is added.

Reply:
I'll be in there soon 😁